clarifai/datasets/upload/zoo/README.md

```diff
@@ -0,0 +1,49 @@
+## Datasets Zoo
```

Comment:
is this really a "zoo" or should we call this "loaders"? zoo/hub is usually a public location where things are shared whereas this folder seems to be more about loading different formats of datasets.

Reply:
Sure. I think we can rename it to "Dataset Loaders" as its more appropriate. Will change `zoo_dataset` to `dataset_loader`